packages/bpk-component-map/src/BpkPriceMarkerV2/BpkPriceMarker.tsx

```diff
@@ -47,7 +47,7 @@ type Props = {
   status?: Status;
 };
 
-const BpkPriceMarkerV2 = (props: Props) => {
+export const BpkPriceMarkerV2 = (props: Props) => {
```

Comment:
for some reason the problem with react docgen was the way the button was exported.. I changed it to a named export because that's what we do for modalv2 and shouldn't matter as in the end the export to consumers stays the same